.travis.yml

```diff
@@ -17,11 +17,14 @@ addons:
       - proj-bin
       - libgeos-dev
       - libopenmpi-dev
+      - gcc-5
 
 env:
   global:
     PIP=pip
     HOMEBREW_NO_AUTO_UPDATE=1
+    CC=gcc-5
```

Comment:
changes in this file are temporary, unless we want to add coverage in CI for older compilers permanently